tests/__init__.py

```diff
@@ -10,26 +11,27 @@
 class BaseSystemTest(unittest.TestCase):
     """Base class for system tests"""
 
-    dynamo: DynamoDBConnection = None
+    dynamo: DynamoDBConnection
+    engine: Engine
 
-    def __init__(self, methodName: str = "runTest") -> None:
-        super().__init__(methodName)
+    @pytest.fixture(autouse=True)
```

Comment:
Replacing `nose test` approach with `pytest` as it seems to be more standard and is supported still.